Tests/MongoSwiftTests/MongoCollection+IndexTests.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+@testable import MongoSwift
+import Nimble
+import NIO
```

Comment:
do we need to import NIO here?

Reply:
wasnt sure so I just copied imports from another file - removing doesnt seem like a concern so am removing